challenge_storing_groceries/src/challenge_storing_groceries/open_door.py

```diff
@@ -6,12 +6,10 @@
 import rospy
 import smach
 import tf2_ros
+# noinspection PyUnresolvedReferences
```

Comment:
Do we recall the reason for this? If not lets remove this.

Reply:
open door needs a lot of work (probably a complete refactor) anyway. So I am fine either way.